### PR TITLE
Overhaul comments for clarity and correctness.

### PR DESCRIPTION
**Pull request by kytrinyx**
Some of the comments were not full sentences ending with a period.
See https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences.

Also, we were a bit inconsistent with the use of language vs track and
exercise vs problem.

### CODE REVIEW

File: paths/paths.go [hunk 1]
```diff
@@ -18,10 +18,10 @@ const (
 )
 
 var (
-	// Home by default will contact the location of your home directory
+	// Home by default will contact the location of your home directory.
 	Home string
 
-	// XdgConfigHome will contain $XDG_CONFIG_HOME if it exists
+	// XdgConfigHome will contain $XDG_CONFIG_HOME if it exists.
```
Comment (Tonkpils) [inline paths/paths.go]: Shouldn't `XDG` be capitalized both in the comment and the constant?

Comment (kytrinyx) [inline paths/paths.go]: Yeah, it sure should. Good catch!

**Comment by kytrinyx** [comment]
Fixed.

**Comment by Tonkpils** [comment]
:+1: Looks great!

**Pull request merged**